ui/cryptomaterial/segmented_control.go

```diff
@@ -71,10 +121,10 @@ func (sc *SegmentedControl) Layout(gtx C) D {
 	)
 }
 
-func (sc *SegmentedControl) TransparentLayout(gtx C) D {
+func (sc *SegmentedControl) splitTileLayout(gtx C) D {
 	sc.handleEvents()
 	return LinearLayout{
-		Width:       gtx.Dp(values.MarginPadding600),
+		Width:       gtx.Dp(values.MarginPadding700),
```

Comment:
Just out if curiosity, any specific reason the width was increased?

Reply:
The width should be increased because if it is only 600, it will limit the information displayed and leave a lot of space on both sides.